bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/IOExceptionWithStatus.java

```diff
@@ -36,6 +37,7 @@ public IOExceptionWithStatus(String message, OperationRuntimeException cause) {
   /**
    * Status from the provided OperationRuntimeException.
    */
+  @SuppressWarnings("deprecation")
```

Comment:
StatusRuntimeException is the correct class to use, and getStatus() is the right method.  I'll explain in person.  This is kind of what we have to do right now.